### PR TITLE
Fixing issue #1318

### PR DESCRIPTION
**Pull request by ymoran00**
Some event object have readonly data property - I've added test for that.
https://github.com/madrobby/zepto/issues/1318

### CODE REVIEW

**Comment by kontrollanten** [comment]
What's the status of this one?

File: src/event.js [hunk 1]
```diff
@@ -65,7 +65,9 @@
       handler.proxy = function(e){
         e = compatible(e)
         if (e.isImmediatePropagationStopped()) return
-        e.data = data
+        var dataPropDescriptor = Object.getOwnPropertyDescriptor(e, 'data')
+        if (!dataPropDescriptor || dataPropDescriptor.writable)
+            e.data = data
```
Comment (kontrollanten) [inline src/event.js]: It seems that when using strict mode `dataPropDescriptor` will be undefined when `e` is `InputEvent` (even though `data` property exists). Seems like a bug to me. To workaround this I would propose to wrap this line around a try/catch and add a comment about this.

**Comment by latel** [comment]
why this fixed is not merged?

**Comment by ymoran00** [comment]
Do I need to do anything? Why isn't it merged already?

**Comment by Kocal** [comment]
Ping @mislav / @madrobby, thanks.

**Pull request closed without merging**